Mathlib/RingTheory/MvPolynomial/MonomialOrder.lean

```diff
@@ -120,6 +163,33 @@ theorem leadingCoeff_monomial {d : σ →₀ ℕ} (c : R) :
 
 @[deprecated (since := "2025-01-31")] alias lCoeff_monomial := leadingCoeff_monomial
 
+@[simp] theorem monic_monomial {d : σ →₀ ℕ} :
+    m.Monic (monomial d (1 : R)) :=
+  m.leadingCoeff_monomial 1
+
+theorem monic_monomial_iff {d : σ →₀ ℕ} {c : R} :
+    m.Monic (monomial d c) ↔ c = 1 :=
```

Comment:
```suggestion
theorem monic_monomial_one {d : σ →₀ ℕ} :
    m.Monic (monomial d (1 : R)) :=
  m.leadingCoeff_monomial 1

@[simp] theorem monic_monomial {d : σ →₀ ℕ} {c : R} :
    m.Monic (monomial d c) ↔ c = 1 :=
```

Reply:
I made both changes, but why the second one?

Reply:
`_iff` should be omitted if unambiguous